news/_posts/2020-09-10-bug-bounty-retrospective.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title:  "Bug Bounty Program, the retrospective"
+subtitle: "When hunters make your business safer!"
+date:   2020-09-11 09:00:00
+authors: [ PierreRambaud, PrestaShop ]
+icon: icon-chat
+image: /assets/images/2020/09/ywhbanner.jpg
+tags: [bug, bounty, program, maintenance]
+---
+
+For several months, we experimented a Bug Bounty Program with the [YesWeHack](https://yeswehack.com) platform. Here is a retrospective, explaining why we did it, and what happened during this period.
+
+
+## Private program
+
+The security of software cannot be assessed unless it is tested by a lot of hunters.
+
+PrestaShop relies on a lot of dependencies, with hundred of thousands lines of code, and this has been true since the very first versions. 
```

Comment:
```suggestion
PrestaShop relies on a lot of dependencies, with hundreds of thousands lines of code, and this has been true since the very first versions. 
```